airlock_processor/BlobCreatedTrigger/__init__.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+import azure.functions as func
+import datetime
+import uuid
+import json
+import re
+
+
+def main(msg: func.ServiceBusMessage,
+         outputEvent: func.Out[func.EventGridOutputEvent]):
+
+    logging.info("Python ServiceBus topic trigger processed message - A new blob was created!.")
+    body = msg.get_body().decode('utf-8')
+    logging.info('Python ServiceBus queue trigger processed message: %s', body)
+
+    json_body = json.loads(body)
+    # message is due to blob creation in an 'in-progress' blob
+    if "stalipim" in json_body["topic"]:
+        completed_step = "submitted"
+        new_status = "in-progress"
+        request_id = re.search(r'/blobServices/default/containers/(.*?)/blobs', json_body["subject"]).group(1)
```

Comment:
is the request_id not in the service bus message, rather than having to find it here?

Reply:
it is not. the reason is that since blob created events are originated from Azure, we can't control the schema.
But we do have the container it was triggered from. This is why I extract this info from there